runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -402,4 +402,7 @@
 	 */
 	@ScriptArguments(integer = 4)
 	public static final int QUEST_FILTER = 3238;
+
+	@ScriptArguments(integer = 1)
```

Comment:
This is not correct. Looking at the script, I can see there's many more than 1 argument:

```
$ rg var_count 5269.rs2asm 
4:.int_var_count      18
5:.string_var_count   1
```